docs/metrics/lead-time-reviews.md

```diff
@@ -1,48 +1,92 @@
--   Exclude pull requests without commits;
--   Exclude push-forced pull requests: where the first commit was authored after the pull request was open.
+-   [Time to open](#time-to-open)
+-   [Time to merge](#time-to-merge)
+
+Pulse aggregates these metrics by week and presents values for the average and the 5%, 25%, 75%, and 95% percentiles.
+
+!!! note
+    Pulse calculates these metrics based on pull request data and:
+
+    -   Includes draft pull requests, meaning that the lifetime of pull requests includes the time marked as draft
+    -   Excludes open and declined pull requests
+    -   Excludes pull requests without commits and force pushed pull requests where the first commit was authored after the pull request was open
 
 ### Time to open
 
-An approximation of the time a change takes to be developed. How long does it take for features to be developed?
+An approximation of how long it takes for a change or feature to be developed:
 
-This metric is calculated as the time between the first commit in the pull request's branch and the pull request being created.
+```text
+pull request open timestamp - first commit in pull request branch timestamp
+```
 
 ### Time to merge
 
-The time spent in the code review process. Once a pull request is opened, how long does it take to merge?
+How long does the code review process take for a change or feature:
 
-This metric is calculated as the time between the pull request being created and the pull request being merged.
+```text
+pull request merge timestamp - pull request open timestamp
+```
 
 ## Review metrics
 
-| Name     | Description                                                                |
-| -------- | -------------------------------------------------------------------------- |
-| Changes  | Number of lines of code changed in a pull request.                         |
-| Commits  | Number of commits in a pull request, even if the pull request is squashed. |
-| Reviews  | Number of reviews in a pull request.                                       |
-| Comments | Number of comments and review comments in a pull request.                  |
+The following metrics provide insights into the level of engagement and efficiency of your code review process:
+
+-   [Time to first review](#time-to-first-review)
+-   [Time to first approval](#time-to-first-approval)
+-   [Time from first to last approval](#time-from-first-to-last-approval)
+-   [Time from last approval to merge](#time-from-last-approval-to-merge)
+-   [Outlier review details](#outlier-review-details)
+
+!!! note
+    Pulse calculates these metrics based on pull request data and:
+
+    -   Considers that reviews are approvals, change requests, and inline pull request comments, but not pull request conversation comments
+    -   Excludes reviews and approvals that are done after the pull requests are merged.
+
+### Time to first review
+
+How long does it take to have the first review on a pull request:
+
+```text
```

Comment:
I've read this a few times, and since I haven't been super involved (yet) in these conversations, everything you wrote made sense. I just have a question these '''texts with the timestamps, are these placeholders? what are you trying to show here? was wondering about that. <3

Reply:
You can [read here](https://github.com/codacy/pulse-user-docs/pull/55#discussion_r635373578) how I'm trying to use the code blocks as a convention for displaying the "formula" that we use to calculate each metric in a systematic way. This is how they will look like on the published documentation:

![image](https://user-images.githubusercontent.com/60105800/119017659-2199f100-b993-11eb-853f-35dd4cc45034.png)

Let me know if this makes more sense now - if not, I already have a couple of ideas on how this can be improved. :+1:

Reply:
Looking good! maybe later we can also collect some feedback on the docs and try to see if these notes are clear. Maybe in the future, we can even add some sort of title per note like "how we calculate these metrics", and then present the calculation. ^_^